### DISCUSSION TITLE
Is the latest version of Subsonic implemented by OpenSubsonic tracked anywhere?

### DISCUSSION THREAD

**Discussion by gabeklavans** [opensubsonic/open-subsonic-api#89] [General]
I noticed some fields missing in OpenSubsonic that are present in the latest Subsonic, such as `songCount` https://www.subsonic.org/pages/api.jsp#search2 in the response of `getAlbumList`.

I saw this when making a client compatible with Gonic, so they're fields that are being used by popular servers that claim to support OpenSubsonic.

I was just wondering if there was any tracking on the latest Subsonic API version that's been implemented, so it can easily be seen when OpenSubsonic has gone out of date, since one of this project's goals is "Maintaining complete backwards compatibility with the existing Subsonic API."

**Comment by gabeklavans** [comment]
So, with my specific example of `songCount`, that might be a different problem with the docs, actually. The example in https://opensubsonic.netlify.app/docs/endpoints/getalbumlist/ shows `songCount` in the return, but navigating the docs from that page -> https://opensubsonic.netlify.app/docs/responses/albumlist/ -> https://opensubsonic.netlify.app/docs/responses/child/ using the links at the bottom I guess gets to the wrong value type.

Anyway, I'm still interested if this version is tracked somewhere.

**Reply by Tolriq** [reply]
There's no possible out of date as Subsonic last version was long before OS was born.
It's usually doc issues as it was a lot of work to rebuild a proper docs, there's also some page I did not migrate and unfortunately no one took the few remaining stuff.

But the code is here you can make PR to fix things.

**Reply by gabeklavans** [reply]
Gotcha, didn't know Subsonic hasn't been updated since.
